transaction-flow.md

```diff
@@ -200,6 +202,18 @@ If a sender has been assigned an `external_id`, this value can be used to find s
   * If you already do KYC on your system, then please contact us and we can waive this requirement from you. In this case you should send us an empty list of documents: `"documents": [ ]` in the request. All of the senders you create in the system will be immediately set to the `approved` state and you won't need to wait for them to get approved.
   * If when creating senders or transactions you get the following error in the response: `"errors":{"documents":[{"error":"blank"}]}` it means that KYC requirements are not yet waived for your account. If we already approved your KYC process and so they should be, then please contact us so we can rectify the issue and update your account accordingly.
   * In case you don't do KYC on your site, then you will need to send us documents that we can use to verify the sender's identity, for more details on this please see the [API reference documentation](https://api.transferzero.com/documentation#documents).
+* Identification number: Identification number of document used
```

Comment:
Since there's a minimum 4-character limit to this, if provided, do we need to mention that here? (It's highly unlikely someone's going to have a 3-digit ID number, but more clarity in documentation seems better to me than the alternative.)

Reply:
Yeah I also thought it's unlikely some's going to have a 3-digit ID number but like you said better being as more specific, it can results in a less query/problem with our customers. I will add it now.